src/js/components/image-fundacion.js

```diff
@@ -0,0 +1,9 @@
+import { newMultimedia, newContainer } from './helpers';
+
+const imageComponet = (data) => {
+  const img = newMultimedia('img', data, ['image-fundation__picture']);
+  const secction = newContainer('section', [img], ['image-fundation']);
```

Comment:
Add the `.container` class to the section so that in desktop it is not full bleed